lib/hanami/action.rb

```diff
@@ -433,7 +433,12 @@ def enforce_accepted_media_types(request)
     # @api private
     def exception_handler(exception)
       config.handled_exceptions.each do |exception_class, handler|
-        return handler if exception.is_a?(exception_class)
+        case exception_class
+        when String
+          return handler if exception.class.name == exception_class # rubocop:disable Style/ClassEqualityComparison
```

Comment:
This was to avoid
```
lib/hanami/action.rb:438:39: C: Style/ClassEqualityComparison: Use instance_of? instead of comparing classes. (https://rubystyle.guide#instance-of-vs-class-comparison)
          return handler if exception.class.name == exception_class
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Which I don't _think_ we want to do, we want to make sure that the two strings match

Reply:
Yeah, I think we actually want to support either a string _or_ a real exception class being provided. Would you mind adjusting it to do that, @mathewdbutton?